versions/3.0.md

```diff
@@ -1128,20 +1128,20 @@ A request body with a referenced model definition.
       "schema": {
         "$ref": "#/components/schemas/User"
       },
-      "examples": [ "http://foo.bar/examples/user-example.json" ]
+      "examples": [ "http://foo.bar/other-api.yaml#/components/examples/user-example.json" ]
```

Comment:
The naming is definitely confusing, to me at least..  `user-example.json` reads like a file name.